src/components/Tabs.jsx

```diff
@@ -1,40 +1,66 @@
 import React, { useState } from 'react';
+import Box from '@material-ui/core/Box';
 import Tabs from '@material-ui/core/Tabs';
 import Tab from '@material-ui/core/Tab';
 import { appleTabsStylesHook } from '@mui-treasury/styles/tabs';
-import Link from '@mui/material/Link';
+import Today from '../components/Today';
+import Yesterday from '../components/Yesterday';
+import Tomorrow from '../components/Tomorrow';
 
 const AppleTabs = () => {
   const [tabIndex, setTabIndex] = useState(1);
   const tabsStyles = appleTabsStylesHook.useTabs();
   const tabItemStyles = appleTabsStylesHook.useTabItem();
 
   return (
-    <Tabs
-      classes={tabsStyles}
-      value={tabIndex}
-      centered
-      selectionFollowsFocus
-      sx={{ 
-        width: '100%',
-        boxShadow: '3px 3px 2px 1px rgba(0, 0, 0, 0.2)',
-        [`& .MuiTypography-root`]: {          
-          color: "#ccc", 
-          textTransform: 'capitalize',       
-          textDecoration: 'none',
-        },
-      }}
-      onChange={(e, index) => setTabIndex(index)}
-      aria-label="Tabs where selection follows "
-    >
-      <Tab classes={tabItemStyles} disableRipple label={<Link href="/yesterday">Yesterday</Link>} />
-      <Tab classes={tabItemStyles} disableRipple label={<Link href="/">Today</Link>} />
-      <Tab classes={tabItemStyles} disableRipple label={<Link href="/tomorrow">Tomorrow</Link>}  />
-    </Tabs>
+    <Box>
+      <Tabs
+        classes={tabsStyles}
+        value={tabIndex}
+        centered
+        selectionFollowsFocus
+        sx={{ 
+          width: '100%',
+          boxShadow: '3px 3px 2px 1px rgba(0, 0, 0, 0.2)',
+          [`& .MuiTypography-root`]: {          
+            textTransform: 'capitalize',       
+            textDecoration: 'none',
+          },
+          [`& .MuiButtonBase-root`]: {
+            color: "#ccc", 
+          }
+        }}
+        onChange={(e, index) => setTabIndex(index)}
+        aria-label="Tabs"
+      >
+        <Tab classes={tabItemStyles} disableRipple label="Yesterday" />
+        <Tab classes={tabItemStyles} disableRipple label="Today" />
+        <Tab classes={tabItemStyles} disableRipple label="Tomorrow"  />
+      </Tabs>
+      <TabPanel value={tabIndex} index={0}>
```

Comment:
Does `tabIndex` correspond to `index`? If so can we do `index={tabIndex}`?

Reply:
No it doesn't corresponds.....

Reply:
Ok